third_party/protobuf/3.4.0/src/google/protobuf/compiler/zip_output_unittest.sh

```diff
@@ -81,7 +81,7 @@ if $JAR c $TEST_TMPDIR/testzip.proto > /dev/null; then
     || fail 'jar failed.'
 
   # Check that -interface.jar timestamps are normalized:
-  if [[ "$(TZ=UTC $JAR tvf $TEST_TMPDIR/testzip.jar)" != *'Tue Jan 01 00:00:00 UTC 1980'* ]]; then
+  if [ "$(TZ=UTC $JAR tvf $TEST_TMPDIR/testzip.jar)" != *'Tue Jan 01 00:00:00 UTC 1980'* ]; then
```

Comment:
This is a bashism, no? The script here is a POSIX shell one.

Reply:
According to [SC2039](https://github.com/koalaman/shellcheck/wiki/SC2039), in POSIX sh, `[[ ]]` is undefined. Since the shebang indicates use of POSIX sh, the previous code should not work at all. Because this particular syntax isn't required, I made it compliant.

Reply:
Ok great, I never know which one is POSIX :)